pkg/controllers/csi/gc/images_test.go

```diff
@@ -165,21 +163,12 @@ func TestCollectUnusedAgentBins(t *testing.T) {
 			db:   metadata.FakeMemoryDB(),
 			path: testPathResolver,
 		}
-		gc.db.InsertDynakube(ctx, &metadata.Dynakube{
-			Name:          "test",
-			TenantUUID:    "test",
-			LatestVersion: "test",
-			ImageDigest:   testImageDigest,
-		})
-		gc.db.InsertVolume(ctx, &metadata.Volume{
-			VolumeID:   "test",
-			TenantUUID: "test",
-			Version:    testVersion1,
-			PodName:    "test",
-		})
-
 		testImageDir := testPathResolver.AgentSharedBinaryDirForAgent(testImageDigest)
 		testZipDir := testPathResolver.AgentSharedBinaryDirForAgent(testVersion1)
+
+		gc.db.CreateCodeModule(ctx, &metadata.CodeModule{Version: testImageDigest, Location: testImageDir})
```

Comment:
```suggestion
		gc.db.CreateCodeModule(ctx, &metadata.CodeModule{Version: "??", Location: testImageDir})
```
typo?

Reply:
Fixed, thanks!
We no longer use image digests

Reply:
Done